docs/signer.md

```diff
@@ -0,0 +1,354 @@
+# Kritis Signer
+
+Kritis Signer is a command-line tool that creates attestations. These attestations can be used for Kritis and Binary Authorization deployment enforcement.
+
+The tool can create attestations directly or conditionally, checking policy conformance based on Vulnerability scanning results from Google Cloud Container Analysis.
```

Comment:
I think vulnerability scanning can be lower case, put the link here to simplify the paragraph that describes how it works.

Reply:
Done.